frontend/src/stores/levels/actions.ts

```diff
@@ -13,4 +13,26 @@ async function getLevelsByRoleId(roleId: number): Promise<Level[] | undefined> {
   return role.levels
 }
 
-export { getLevelsByRoleId }
+async function getScoreResponse(
+  roleId: number
+): Promise<{ currentLevel: Level; nextLevel: Level } | undefined> {
+  const answers = [
```

Comment:
This object will not be mounted here. It will be mounted in the ResultScreenView